openid-4-verifiable-presentations-1_0.md

```diff
@@ -574,7 +574,7 @@ Body
 
 * `x509_san_dns`: When the Client Identifier Scheme is `x509_san_dns`, the Client Identifier MUST be a DNS name and match a `dNSName` Subject Alternative Name (SAN) [@!RFC5280] entry in the leaf certificate passed with the request. The request MUST be signed with the private key corresponding to the public key in the leaf X.509 certificate of the certificate chain added to the request in the `x5c` JOSE header [@!RFC7515] of the signed request object. The Wallet MUST validate the signature and the trust chain of the X.509 certificate. All Verifier metadata other than the public key MUST be obtained from the `client_metadata` parameter. If the Wallet can establish trust in the Client Identifier authenticated through the certificate, e.g. because the Client Identifier is contained in a list of trusted Client Identifiers, it may allow the client to freely choose the `redirect_uri` value. If not, the FQDN of the `redirect_uri` value MUST match the Client Identifier without the prefix `x509_san_dns:`. Example Client Identifier: `x509_san_dns:client.example.org`.
 
-* `web-origin`: This Client Identifier Scheme is defined in (#dc_api_request). The Wallet MUST NOT accept this Client Identifier Scheme if the request is not sent via the Digital Credentials API defined in (#dc_api).
+* `origin`: This Client Identifier Scheme is defined in (#dc_api_request). The Wallet MUST NOT accept this Client Identifier Scheme in requests. In OpenID4VP over the Digital Credentials API, the audience of the Credential Presentation is always the origin value prefixed by `origin:`, for example `origin:https://verifier.example.com/`.
```

Comment:
Do we still need this scheme?

Reply:
I brought up the same question before (https://github.com/openid/OpenID4VP/pull/448#discussion_r1984999561) and this seems to be more of a placeholder so nobody use that as a scheme. It is technically not a real client id scheme (cannot be used in the request), but blocked for the response.

I don't really like it but I guess this is fine?

Reply:
Yes, what Christian said - client id schemes are basically defining prefixes to be used in two places:

1. The client id parameter
2. The `aud` value in responses

The `origin:` prefix is definitely used in "2" so I think to avoid any potential for future confusion it should be a "registered" client id scheme.